pybikes/smoove.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2015, Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import re
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils, exceptions
+
+__all__ = ['Smoove', 'SmooveStation']
+
+# Each station is formatted as:
+# newmark_01(28, 45.770958,3.073871, "<div class=\"mapbal\" align=\"left\">022 Jaurès<br>Vélos disponibles: 4<br>Emplacements libres: 10<br>CB: Non<br></div>");
+# I.e., (station_id, latitude, longitude, name, available_bikes, free_bike_stands, credit_card_enabled (discarded afterwards))
+DATA_RGX = r'\(\d+\,\ (\d+.\d+).*?(\d+.\d+)\,\ "<div.*?>(.*?)<br>.*?:(.*?)<br>.*?:(.*?)<br>.*?:(.*?)<br></div>"\)'
```

Comment:
I did that, looks uglier than with regex only because the html that describes the station status is not structure, it is only <br> (not even terminated one) separated.

Take a look.